src/client/app/App.jsx

```diff
@@ -10,6 +10,8 @@ import NotFound from './components/NotFound';
 import Dashboard from './components/dashboard/Dashboard';
 import Calendar from './components/calendar/Calendar';
 import AddKiddo from './components/add-child/AddKiddo';
+import KiddoProfile from './components/kiddo-profile/KiddoProfile';
+import Map from './components/map/Map';
```

Comment:
Map is a protected keyword
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map